puppet/modules/jenkins_job_builder/files/centos.org/ansible/fetch_debug_files.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: any
+  tasks:
+    - name: 'Find any files from debug'
+      find:
+        paths: './forklift/pipelines/debug'
```

Comment:
Should this be a hard-coded or computed path rather than relative?

Reply:
Especially given this path -- https://github.com/theforeman/foreman-infra/pull/687/files#diff-0c0d3261627c35f21d3c6d6f54c399daR12

Reply:
yeah, I'll go ahead and hard-code it.